pkg/apis/kudo/v1beta1/instance_types_helpers.go

```diff
@@ -90,3 +94,47 @@ func wasRunAfter(p1 PlanStatus, p2 PlanStatus) bool {
 	}
 	return p1.LastFinishedRun.Time.After(p2.LastFinishedRun.Time)
 }
+
+// GetParamDefinitions retrieves parameter metadata from OperatorVersion CRD
+func GetParamDefinitions(params map[string]string, ov *OperatorVersion) []Parameter {
```

Comment:
These were simply moved from the `instance_controller.go` since they're now used in two places.